pkg/cmd/roachtest/engine_switch.go

```diff
@@ -140,6 +140,7 @@ func registerEngineSwitch(r *testRegistry) {
 	r.Add(testSpec{
 		Name:       fmt.Sprintf("engine/switch/nodes=%d", n),
 		Owner:      OwnerStorage,
+		Skip:       "rocksdb removed in 21.1",
```

Comment:
since roachtest is always based on master, doesn't this block all invocations?

Reply:
It does. I'm told @jlinder+@adityamaru are working to address https://github.com/cockroachdb/cockroach/issues/51897 shortly.